frontend/packages/dev-console/src/components/import/DeployImage.tsx

```diff
@@ -6,6 +6,7 @@ import { DeployImageFormData } from './import-types';
 import { createResources } from './deployImage-submit-utils';
 import { deployValidationSchema } from './deployImage-validation-utils';
 import DeployImageForm from './DeployImageForm';
+import { createKnativeService } from '../../utils/create-knative-utils';
```

Comment:
PR #1800 will move knative specific resources to knative-plugin. It's good for now. If this PR goes in i'll do the rebase and update

Reply:
Whoever merges first gets to rebase/refactor. ;-)